mava/systems/sable/anakin/ff_sable.py

```diff
@@ -372,17 +372,13 @@ def learner_setup(
     # Get available TPU cores.
     n_devices = len(jax.devices())
 
-    # Get number of agents.
-    config.system.num_agents = env.num_agents
-
     # PRNG keys.
     key, net_key = keys
 
     # Get number of agents and actions.
     action_dim = env.action_dim
-    n_agents = env.action_spec().shape[0]
+    n_agents = env.num_agents
     config.system.num_agents = n_agents
-    config.system.num_actions = action_dim
```

Comment:
Why remove this from the config?

Reply:
This was from a copy of old ppo but now since we call `action_dim` only in the `learner_setup` and we're not using it in other parts of code like the `num_agents` I removed it from here